taskt/Core/Script/Script.cs

```diff
@@ -0,0 +1,181 @@
+﻿//Copyright (c) 2019 Jason Bayldon
+//
+//Licensed under the Apache License, Version 2.0 (the "License");
+//you may not use this file except in compliance with the License.
+//You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+//Unless required by applicable law or agreed to in writing, software
+//distributed under the License is distributed on an "AS IS" BASIS,
+//WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//See the License for the specific language governing permissions and
+//limitations under the License.
+namespace taskt.Core.Script
+{
+    using System.Collections.Generic;
```

Comment:
To my knowledge, using should always go on top in C#

Reply:
Made the changes, give them a quick lookover.

Reply:
Looks good. Much easier to read.